apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -157,7 +157,15 @@ function getStandardizedAliases(metadata) {
 function getStandardizedCategories(metadata) {
   // If the animation doesn't have a category, place it in a section for
   // level-specific/hidden-from-library animations.
-  return metadata.categories || ['level_animations'];
+  // Animations that are level-specific and NOT backgrounds are costumes and
+  // either do not have the `categories` property or have categories assigned to [""]
+  const categories = metadata.categories;
+  const isLevelAnimationCostume =
+    categories === undefined || categories[0] === '';
+  if (isLevelAnimationCostume) {
+    return ['level_animations'];
```

Comment:
constant for `'level_animations'` since it's referenced so much?

Reply:
Also, to check my understanding - "level_animations" refers to level-specific animations that are not backgrounds, right? In which case would "level_costumes" or something be clearer? I think i'm getting a bit tripped up on the naming and the distinction between these different types.

Reply:
Yes - 'level_animations' refers to level-specific animations that are not backgrounds.
Currently, the image path is assigned in `SpriteUpload.jsx` [here](https://github.com/code-dot-org/code-dot-org/blob/11012fd30f01eae878ae3e57f8007fa7f34e6627/apps/src/code-studio/assets/SpriteUpload.jsx#L97).
The animation JSON category value is assigned [here](https://github.com/code-dot-org/code-dot-org/blob/11012fd30f01eae878ae3e57f8007fa7f34e6627/apps/src/code-studio/assets/SpriteUpload.jsx#L525).
But I updated the category which is used for the manifest to 'level_costumes' in `animationLibraryApi.js` with https://github.com/code-dot-org/code-dot-org/pull/55333/commits/c3d75cf8babd8d633f4b112f04d2f1e706407814. Thanks!